common/src/main/kotlin/io/homeassistant/companion/android/common/compose/theme/HATheme.kt

```diff
@@ -29,3 +34,15 @@ fun HATheme(darkTheme: Boolean = isSystemInDarkTheme(), content: @Composable ()
         )
     }
 }
+
+/**
+ * Small Wrapper around [HATheme] to handle the background color for screenshots/preview, to test individual screen we
+ * need to set the background color manually.
```

Comment:
> to test individual screen we need to set the background color manually

I think I know what you mean because of the PR description, but not when reading this description. Suggestion to simplify it:

```suggestion
 * Small wrapper around [HATheme] for previews/screenshot tests which:
 * - applies the theme
 * - adds a container with a background color around the content
```